composer.json

```
@@ -5,12 +5,12 @@
     "homepage": "https://www.silverstripe.org",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-cms": "4.x-dev",
-        "dnadesign/silverstripe-elemental": "4.x-dev",
-        "silverstripe/elemental-fileblock": "2.x-dev",
-        "silverstripe/elemental-bannerblock": "2.x-dev"
+        "php": "^8.1",
+        "silverstripe/recipe-plugin": "^2",
+        "silverstripe/recipe-cms": "5.x-dev",
+        "dnadesign/silverstripe-elemental": "5.0.0-alpha2",
```

Most of the dependencies in recipes need to be targeted to a `.x-dev` branch. When COW tag the recipe it rewrites those dependencies.

```suggestion
        "dnadesign/silverstripe-elemental": "5.x-dev",
```

---

DONE